CRP.Core/Domain/FinancialAccount.hbm.xml

```diff
@@ -15,7 +15,7 @@
     <property name="SubAccount" column="SubAccount" />
     <property name="Project" column="Project" />
 
-    <many-to-one name="Unit" class="CRP.Core.Domain.Unit, CRP.Core" column="UnitId" cascade="none" not-null="true" fetch="join" />
+    <many-to-one name="Unit" class="CRP.Core.Domain.Unit, CRP.Core" column="UnitId" cascade="none" fetch="join" />
```

Comment:
@srkirkland This was the other change I made that I wasn't a 100% sure about (and the related DB change).
I'm guessing John was thinking for a future version when users would pick their own accounts for their events instead of picking from a list (or to filter that list to their unit).
But as of now, only our accounting people can add accounts and the page didn't have a way to set the unit.